samcli/commands/local/cli_common/options.py

```diff
@@ -48,7 +48,16 @@ def local_common_options(f):
             default=False,
             help="If set, will emulate a shutdown event after the invoke completes, "
             "in order to test extension handling of shutdown behavior.",
-        )
+        ),
+        click.option(
+            "--container-host",
+            default="localhost",
+            show_default=True,
+            help="Host of locally emulated Lambda container. "
+            "This option is useful when the containers run on a different host than SAM CLI. "
+            "For example, if you want to run SAM CLI in a Docker container on macOS, "
+            "use this option with host.docker.internal",
```

Comment:
This is me being pedantic, but all help descriptions seem to be full sentences. I wonder if this looks more coherent...

```suggestion
            "use this option with host.docker.internal.",
```

Reply:
Eh never mind, we have some inconsistency elsewhere. I can do a full pass of all help messages later to satisfy my pedantic whims.